src/frequenz/sdk/actor/power_distributing/_distribution_algorithm/_battery_distribution_algorithm.py

```diff
@@ -685,7 +685,16 @@ def distribute_power(
         Returns:
             Distribution result
         """
-        if power >= 0.0:
+        if is_close_to_zero(power):
+            return DistributionResult(
+                distribution={
+                    inverter.component_id: 0.0
+                    for _, inverters in components
+                    for inverter in inverters
+                },
+                remaining_power=0.0,
+            )
```

Comment:
In this case, if I understand correctly, you are setting it to zero if it is close to zero, right?

Reply:
yup, this is the value that gets sent to the microgrid API.